bin/misc-functions.sh

```diff
@@ -218,12 +218,16 @@ install_qa_check() {
 		)
 	done < <(printf "%s\0" "${qa_checks[@]}" | LC_ALL=C sort -u -z)
 
-	export STRIP_MASK
+	export STRIP
```

Comment:
I don't think I did that. Wouldn't this break some of the ebuilds relying on STRIP_MASK?

Reply:
Rebase error, fixed now.